doc/source/whatsnew/v0.25.1.rst

```diff
@@ -7,6 +7,18 @@
 What's new in 0.25.1 (July XX, 2019)
 ------------------------------------
 
+.. warning::
```

Comment:
This should go to 1.0. We don't add deprecations in point releases.

There should be "Deprecations" section.

Don't need the warning boxes.

Do need a link back to the issue number with :issue:

Reply:
Thank you, moved it to v1.0.0.rst